### PR TITLE
Support coco panoptic annotations in create_coco_tf_record.py

### PR DESCRIPTION
**Pull request by srihari-humbarwadi**
# Description
Extends existing `create_coco_tf_record.py` script to load and export panoptic annotations in tfrecord files

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have signed the [Contributor License Agreement](https://github.com/tensorflow/models/wiki/Contributor-License-Agreements).
- [x] I have read [guidelines for pull request](https://github.com/tensorflow/models/wiki/Submitting-a-pull-request).
- [x] My code follows the [coding guidelines](https://github.com/tensorflow/models/wiki/Coding-guidelines).
- [x] I have performed a self [code review](https://github.com/tensorflow/models/wiki/Code-review) of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: official/vision/beta/data/create_coco_tf_record.py [hunk 1]
```diff
@@ -58,6 +57,10 @@
     'annotations - boxes and instance masks.')
 flags.DEFINE_string('caption_annotations_file', '', 'File containing image '
                     'captions.')
+flags.DEFINE_string('panoptic_annotations_file', '', 'File containing panoptic '
+                    'annotations.')
+flags.DEFINE_string('panoptic_masks_dir', '', 'Directory containing panoptic '
```
[the remark this answers is not in the document]
Comment (arashwan) [reply official/vision/beta/data/create_coco_tf_record.py:62]: Can you add "Directory containing panoptic masks annotations".

File: official/vision/beta/projects/panoptic_maskrcnn/dataloaders/panoptic_maskrcnn_input.py [hunk 2]
```diff
@@ -24,25 +24,51 @@
 class TfExampleDecoder(tf_example_decoder.TfExampleDecoder):
   """Tensorflow Example proto decoder."""
 
-  def __init__(self, regenerate_source_id, mask_binarize_threshold):
+  def __init__(self, regenerate_source_id, 
+               mask_binarize_threshold, include_eval_masks):
```
Comment (arashwan) [inline official/vision/beta/projects/panoptic_maskrcnn/dataloaders/panoptic_maskrcnn_input.py:28]: maybe include_panoptic_masks is a better naming?

File: official/vision/beta/data/create_coco_tf_record.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (srihari-humbarwadi) [reply official/vision/beta/data/create_coco_tf_record.py]: coco panoptic dataset comes with a folder containing panoptic annotations as masks in PNG format. We transform these panoptic masks into
 - segmentation targets
 - category_mask and instance_mask for PQ evaluator

More about [this](https://cocodataset.org/#format-data), under Panoptic Segmentation

**Review by arashwan — APPROVED** [review]

**Review by jaeyounkim — APPROVED** [review]
